marconi-chain-index/test-lib/Gen/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -136,29 +76,14 @@ utxoAddressOverride
   -> Utxo
 utxoAddressOverride addr utxo =  utxo {_address=C.toAddressAny addr}
 
--- | Generate ShelleyEra Utxo Events
-genShelleyEraUtxoEvents :: Gen [StorableEvent UtxoHandle]
+-- | generate Shelley Era Utxos
```

Comment:
Why did the comment change?

Reply:
Fixed.  It was my error as I had removed some code and this part stayed form the old code.

Reply:
I had re-arranged parts of this module and mixed this line with another function that I had taken out. It is fixed now.